app/views/visns/_show.js.erb

```diff
@@ -417,4 +432,14 @@ function execVisnSearchFunctions() {
 document.addEventListener('turbolinks:load', function() {
     execVisnSearchFunctions();
     submitFormOnPageLoadWithCreatedPracticesAsDefault();
-});
\ No newline at end of file
+});
+
+function updateSelectedCategoriesUsage(sQuery, chosenCategories, category_id){
```

Comment:
I think we should move this into a separate `es6` file that we load in  since this fn is being used in different js files, right?

if we do that we can make use of this: Javascript's (I believe it might be specific to es6 or something like that) destructuring and setting default values so we don't have to pass in `null` values and it is more clear what we are passing to the function
example: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment
```
 function drawChart({size = 'big', coords = {x: 0, y: 0}, radius = 25} = {}) {
  console.log(size, coords, radius);
  // do some chart drawing
}

drawChart({
  coords: {x: 18, y: 30},
  radius: 30
});
```

Reply:
Added this method to it's own es6 file and added to assets pipeline and application.js as well.